jobs/import-to-table-partition.yml

```diff
@@ -14,6 +15,7 @@ configuration:
     datasource: "openmrs-${siteName}.yml"
     query:  "sql/extractions/${tableName}.sql"
     context: "sql/utils/context.sql"
+    conditional: "${conditional}"

```

Comment:
This allows the import-to-table-partition job to take in a conditional as an parameter to determine if it should run.